src/controller/CHIPDeviceController.cpp

```diff
@@ -597,6 +597,12 @@ CHIP_ERROR DeviceCommissioner::PairDevice(NodeId remoteDeviceId, RendezvousParam
         }
 #endif // CONFIG_DEVICE_LAYER && CONFIG_NETWORK_LAYER_BLE
     }
+    else if (params.GetPeerAddress().GetTransportType() == Transport::Type::kTcp ||
+             params.GetPeerAddress().GetTransportType() == Transport::Type::kUdp)
+    {
+        udpPeerAddress = Transport::PeerAddress::UDP(params.GetPeerAddress().GetIPAddress(), params.GetPeerAddress().GetPort(),
```

Comment:
So this makes clear something that was implicit: even if the peer address here is TCP, we're going to do UDP.  Is there a followup issue tracking sorting this out?

Reply:
we have no CASE support currently, so tcp is not used.

I am unclear about long term - it seems to me like most communication would be UDP anyway because battery powered devices really do not like TCP.

I did not create a issue tracking this because I cannoto express in words what the expected final result should be. I expect this will correct itself as we introduce case support and we look into things that may actually want TCP (in this case OTA upgrades)

Reply:
@pan-apple - any suggestions on plans here? Is this something that should be specifically tracked or we ignore for now as we will address things as we develop further?

Reply:
@andy31415 We don't need to know what the right solution is to track a problem....  I filed https://github.com/project-chip/connectedhomeip/issues/5128